scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
No scripts/. And definitely no scripts/helpers, scripts/ contains too much stuff for this name to be useful: this is helpers to what? Generating C functions’ declarations? Generating Ex commands lists, eval functions or options? Generating Vim syntax files?

Or maybe even running PVS checks as `helpers.lua` may just as well be a helpers script used like `lua helpers.lua --do-some-stuff`?